### DISCUSSION TITLE
Out dir not working as intended

### DISCUSSION THREAD

**Discussion by TakenMC** [swc-project/swc#9479] [Q&A]
When i used tsc previously I would have it transpile the src to dist

For some reason when I use swc `swc src -d dist` it would send the output to dist/src instead of dist which bugs me a lot since I don't know if I am missing something or this is a bug.

Anyone know how I could fix this issue?

**Comment by TakenMC** [comment]
https://github.com/swc-project/swc/discussions/8644

**Comment by kdy1** [comment]
You can try `--strip-leading-paths`
